### ISSUE TITLE
Don't parse escaped annotations within comments on methods and variables

### ISSUE THREAD

**Issue by pozil** [cesarParra/apexdocs#93]
Hi @cesarParra this issue is a follow up on #88. I just realized that while you deployed a fix for escaped annotations within class-level comments, the issue persists on method/variable comments.

For example, generating doc for the following code:
```apex
/**
 * @description Demonstrates the `@future` syntax and usage
 * @group Async Apex Recipes
 */
public with sharing class AtFutureRecipes {
    /**
     * This class features a 'circuit breaker' variable. While this is not a
     * pattern I routinely turn to, it is a valuable pattern for testing code
     * that is otherwise hard to test. For instance, `@future` methods must be
     * static void methods...
     */
    @testVisible
    private static Boolean testCircuitBreaker = false;
}
```

will crash with this error:
```
10/4/2023, 4:25:35 PM: AtFutureRecipes - Doc comment parsing error. Level: Field
10/4/2023, 4:25:35 PM: Comment:
 /**
   * This class features a 'circuit breaker' variable. While this is not a
   * pattern I routinely turn to, it is a valuable pattern for testing code
   * that is otherwise hard to test. For instance, `@future` methods must be
   * static void methods...
   */
10/4/2023, 4:25:35 PM: Exception: Error parsing Apex doc. Line 4:53 - mismatched input '`@future`' expecting {NAME, NEWLINE, SPACE, TEXT_CONTENT, '*', '/', JAVADOC_END, '{@', '{', '}'}
```

**Comment by cesarParra** [comment]
I have identified the issue. The gap is with description blocks without the preceding `@description` tag. Working on a fix.

**Comment by pozil** [comment]
Awesome, thanks a lot. I'll add the description tag in the meantime.

**Comment by cesarParra** [comment]
Addressed in v2.17.1 https://www.npmjs.com/package/@cparra/apexdocs/v/2.17.1
